src/api/api_auth_shibuqam.nit

```diff
@@ -94,7 +99,10 @@ class ShibCallback
 			config.players.save player
 		end
 		session.player = player
-		res.redirect "/player"
+
+		# Redirect
+		var page = session.auth_next or else "/player"
```

Comment:
If I introduce a common abstract superclass in api_auth with the common services/code, is that OK for you?